packages/cxl-ui/scss/global/cxl-section.scss

```diff
@@ -28,4 +28,12 @@ cxl-section#view-hero, cxl-section[theme~="cxl-hero"] {
     }
   }
 
+  h1 {
+    font-size: var(--lumo-font-size-xxl);
+
+    @media #{mq.$medium} {
+      font-size: calc(var(--lumo-font-size-xxl) + var(--lumo-font-size-xxxl) * 2/3);
```

Comment:
wow, this is complex. no way of making it simpler? ;)

Reply:
Current size is 54px; The largest lumo style is 40px; cxl-hero-size is 64px; This was the closest I could get it to the current size(58px) using only lumo styles.

Reply:
I'm missing something here, give me a minute.